scripts/xspdb/README.en.md

```diff
@@ -0,0 +1,194 @@
+## Introduction to XSPdb
+
+[中文介绍](/README.cn.md)
+
+XSPdb is a specialized Python pdb-based debugging tool for RISC-V IP cores, customized for Xiangshan's difftest interface. It provides GDB-like interactive debugging capabilities, integrating: Terminal command-line interface, RTL-level waveform toggling, Automated script replay, System snapshot save/restore, Register initialization configuration, Instruction set disassembly, etc. Advanced debugging features: conditional breakpoints, real-time watchpoints, register/memory visualization, Hardware signal-level debugging interface synchronized with software execution states. It is a Hardware/software co-verification solution for RISC-V IPs.
```

Comment:
add a short desc and quickstart about XSPdb in root README.md ( inside `Run Programs by Simulation` section, after `Run with simulator`), and add a link to this full md